enderio-base/src/main/java/com/enderio/base/common/util/ItemModelUtils.java

```diff
@@ -19,42 +19,44 @@ public static ItemModelBuilder fakeBlockModel(DataGenContext<Item, ? extends Ite
     public static ItemModelBuilder mimicItem(DataGenContext<Item, ? extends Item> ctx, ItemEntry<? extends Item> item, RegistrateItemModelProvider prov) {
         return prov.generated(ctx, prov.itemTexture(item));
     }
-    
+
     public static void gearItem(DataGenContext<Item, ? extends Item> ctx, RegistrateItemModelProvider prov) {
         // json so the BEWLR is used + perspectives
-        prov.getBuilder(ctx.get().getRegistryName().getPath().toString())
-                .parent((new ModelFile.UncheckedModelFile("builtin/entity")))
-                .transforms()
-                    .transform(Perspective.GROUND)
-                    .rotation(0, 0, 0)
-                    .translation(0, 2, 0)
-                    .scale(0.5F, 0.5F, 0.5F)
-                    .end()
-                .transform(Perspective.HEAD)
-                    .rotation(0, 180, 0)
-                    .translation(0, 13, 7)
-                    .scale(1F, 1F, 1F)
-                    .end()
-                .transform(Perspective.THIRDPERSON_RIGHT)
-                    .rotation(0, 0, 0)
-                    .translation(0, 3, 1)
-                    .scale(0.55F, 0.55F, 0.55F)
-                    .end()
-                .transform(Perspective.FIRSTPERSON_RIGHT)
-                    .rotation(0, -90F, 25F)
-                    .translation(1.13F, 3.2F, 1.13F)
-                    .scale(0.68F, 0.68F, 0.68F)
-                    .end()
-                .transform(Perspective.FIXED)
-                    .rotation(0, 180, 0)
-                    .translation(0, 0, 0)
-                    .scale(1F, 1F, 1F)
-                    .end()
-                .end();
+        prov
+            .getBuilder(ctx.get().getRegistryName().getPath().toString())
+            .parent((new ModelFile.UncheckedModelFile("builtin/entity")))
+            .transforms()
+            .transform(Perspective.GROUND)
+            .rotation(0, 0, 0)
+            .translation(0, 2, 0)
+            .scale(0.5F, 0.5F, 0.5F)
+            .end()
```

Comment:
merge each transform till the end method call to a line and disable the formatter similar to the PaintedModelBuilder